shepherd/lowcar/devices/Arduino4/Arduino4.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
```

Comment:
remove

Reply:
the whole file, or just the pragma once?